code/obj/storage/secure_closets.dm

```diff
@@ -175,6 +175,8 @@
 	name = "\improper Security equipment locker"
 	spawn_contents = list(/obj/item/storage/box/clothing/security,
 	/obj/item/clothing/suit/wintercoat/security,
+	/obj/item/clothing/head/helmet/space/security,
```

Comment:
There is no reason for the space-suit to be on the ocean maps.

I recommend adding these suits manually to each map-file or finding a storage object that you can only find on the space-maps to modify.

Reply:
That is true, I should do that.